pkgs/development/python-modules/typedload/default.nix

```
@@ -0,0 +1,43 @@
+{
+  attrs,
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pytestCheckHook,
+  python,
+  setuptools,
+}:
+
+let
+  typedload = buildPythonPackage rec {
+    pname = "typedload";
+    version = "2.37";
+    format = "pyproject";
```

This has been replaced with:
```suggestion
    pyproject = true;
```

---

Fixed in 0f9707085d655e1fc51539fb1f0264e5ac48767d; thanks!